tests/foreman/ui/test_user.py

```diff
@@ -17,8 +17,10 @@
 """
 
 import random
+
 from fauxfactory import gen_string
 from nailgun import entities
+
```

Comment:
Yeah, Its fine like here:  https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui/test_repository.py#L19-to-L23